src/darsia/utils/linear_solvers/jacobi.py

```diff
@@ -21,7 +21,9 @@ class Jacobi(da.Solver):
 
     """
 
-    def _neighbor_accumulation(self, im: np.ndarray) -> np.ndarray:
+    def _neighbor_accumulation(
+        self, im: np.ndarray, h: Optional[Union[float, list]] = None
+    ) -> np.ndarray:
         """Accumulation of neighbor pixels.
```

Comment:
Better documentation would be good here. What is "accumulation" and how does it depend on the grid size provided?